template/src/ontology/Makefile.jinja2

```diff
@@ -1556,6 +1556,28 @@ update_docs:
 	mkdocs gh-deploy --config-file ../../mkdocs.yaml
 {%- endif %}
 
+.PHONY: release_changelog
+release_changelog: $(REPORTDIR)/changelog.md $(REPORTDIR)/changelog.csv $(REPORTDIR)/changelog.yaml
+
+$(TMPDIR)/latest_release.obo: $(TMPDIR)
+	wget $(ONTBASE)/$(ONT)-base.owl -O $@.tmp.owl || { \
```

Comment:
I wonder if the dependence on "base" here should be documented somewhere. Not everyone likes the idea of having a base, but everyone will want to generate release notes.. Can we make the "diff" ontology configurable, using base by default if and only if it is a configured release product, otherwise the main release?